small-talk/app/chat/server.js

```diff
@@ -0,0 +1,15 @@
+const app = require('./app');
+
+app.listen(8000, ()=>{
+    console.log("Server listening on port 8000");
+})
+
+const io = require("socket.io")(serverm, {
+    cors: {
+        origin: true,
+        methods: ['GET', 'POST'],
+        credentials: true
+    }
+});
+
+io.on("connection", )
```

Comment:
Is this unfinished?

Reply:
Oh I forgot this was the first baseline when we first started working and should have deleted it. Thank you and good catch.

Reply:
Pushed the newer version just now